src/Microsoft.Identity.Client/Cache/Keys/MsalAccessTokenCacheKey.cs

```diff
@@ -65,5 +67,28 @@ public override string ToString()
         public int iOSType => (int)MsalCacheKeys.iOSCredentialAttrType.AccessToken;
 
         #endregion
+
+        #region Equals and GetHashCode
```

Comment:
>#region [](start = 8, length = 7)

nit: personally, don't think we need the region here...

Reply:
I like regions, they group similar functionality. In this particular case, it's very important that any changes to Equals are made to GetHashCode, otherwise if you put those obects in a dictionary it will behave in strange ways.

Reply:
I'm not saying I want to refactor the entire codebase and add regions, but large files can benefit from them.